e2e/terraform/terraform.tfvars

```diff
@@ -1,10 +1,14 @@
 region               = "us-east-1"
```

Comment:
Note for reviewers: we're seeing a diff here, but this is just the `terraform.tfvars.dev` file moved to `terraform.tfvars`, and the old `terraform.tfvars` moved to `terraform.full.tfvars`. The vars files are otherwise unedited.